setup.py

```diff
@@ -52,7 +52,7 @@
         'numpy>=1.20',
         'psutil',
         'pyradiosky>=0.2.0',
-        'pyuvdata>=2.2.10',
+        'pyuvdata>=2.2.12',
```

Comment:
I think we should consider upgrading to require v2.4.1 so that if the sims are saved out in files, the frame information is preserved.